src/pages/terraform/internals/[[...page]].tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react'
+import terraformData from 'data/terraform.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+
+const basePath = 'internals'
+const baseName = 'Internals'
+const product = terraformData as Product
+// TODO: productSlug should possibly actually be `terraform`,
+// TODO: but https://mktg-content-api.vercel.app/api/content/terraform/version-metadata?partial=true
+// TODO: does not return any version with "isLatest: true"
+const productSlugForLoader = 'terraform-website'
+
+const additionalComponents = {}
```

Comment:
Same question as previous comments about this, just commenting so it's identified in case you end up making changes! 💖